ui/lib/replication/addon/templates/components/replication-primary-card.hbs

```diff
@@ -5,9 +5,10 @@
       {{description}}
     </span>
     <h3 class="title is-3">
-      {{!-- TODO make this dynamic --}}
-      <Icon @glyph="check-circle-outline" class="has-text-success" />
       {{metric}}
+      {{#if glyph}}
+        <Icon @glyph={{glyph}} class="has-text-success" />
```

Comment:
Will the state always be a positive/successful one?

Reply:
excellent eye! yes, we only ever show a glyph if the state is positive. you can see where we get the glyph from the cluster model [here](https://github.com/hashicorp/vault/blob/c5efd226eda702217b5b18edcf42cc215e9e32eb/ui/app/models/cluster.js#L77).

if the cluster is in a bad state will show an error alert banner, but it'll be above the state. i'm working on that in the next PR 👯